### PR TITLE
fix linter errors in 0001-UC-lifecycle

### PR DESCRIPTION
**Pull request by lixuna**
- removing line 5 fixes `5 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]`
- adding line before line 35 fixes `35 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: “- one fresh node gets created ...“]`
- removing trailing space in line 39 fixes `39:195 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`
- removing trailing space in line 41 fixes `41:548 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`
- removing trailing space in line 45 fixes `45:21 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`
- removing trailing space in line 61 fixes `61:86 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]`

### CODE REVIEW

**Comment by electrocucaracha** [comment]
Apparently [the line 9 of the glossary file](https://github.com/cncf/cnf-wg/pull/170/checks?check_run_id=2577050483#step:4:201) has also an extra space

**Comment by lixuna** [comment]
@electrocucaracha, re: Apparently the line 9 of the glossary file has also an extra space
- please see https://github.com/cncf/cnf-wg/pull/169#discussion_r631927135

File: use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md [hunk 1]
```diff
@@ -2,12 +2,10 @@
 
 ## Glossary
 
-
 - CNF - Cloud Native Network Function
 - Cloud Native Platform - A platform that exhibits cloud like properties on which a CNF runs. For this use case, Cloud Native Platform = Kubernetes cluster
 
 ## Involved processes
-
```
Comment (electrocucaracha) [inline use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md]: This extra line is needed, maybe cherry-picking the second commit can fix the issues

File: use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md [hunk 2]
```diff
@@ -2,12 +2,10 @@
 
 ## Glossary
 
-
 - CNF - Cloud Native Network Function
 - Cloud Native Platform - A platform that exhibits cloud like properties on which a CNF runs. For this use case, Cloud Native Platform = Kubernetes cluster
 
 ## Involved processes
```
Comment (taylor) [inline use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md]:
```suggestion
## Involved processes

```

File: use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md (hunk 1 above)
Comment (taylor) [reply use-case/0001-UC-lifecycle-of-infrastructure-where-CNF-is-running.md]: @electrocucaracha I've added it back

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Pull request merged**